src/commonMain/kotlin/exchange.dydx.abacus/state/internalstate/InternalState.kt

```diff
@@ -173,6 +173,10 @@ internal data class InternalTradeInputOptions(
 
 internal data class InternalMarketSummaryState(
     var markets: MutableMap<String, InternalMarketState> = mutableMapOf(),
+
+    var volume24HUSDC: Double? = null,
+    var openInterestUSDC: Double? = null,
+    var trades24H: Double? = null,
```

Comment:
these seem like they could default to 0 and be non-nullable

Reply:
There is a difference though.  null means the value is pending